src/shaders/triangle.vert

```diff
@@ -0,0 +1,6 @@
+layout(location = 0) in highp vec4 position;
+out int dummyvar;
+
+void main() {
+    dummyvar = 4;
```

Comment:
The shader doesn't write to `gl_Position` so the GPU discards everything, and that could explain why you get nothing on the output. It needs to multiply the position with transformation and projection matrix, like the other shaders do.

Reply:
Is it possible to just have the fragment shader? I only need gl_PrimitiveID, but I added a vertex shader because I didn't know if both were necessary